packages/shared/components/atoms/index.js

```diff
@@ -1,4 +1,6 @@
 export * from './pills'
 export * from './boxes'
+export * from './buttons'
 export { default as MenuButton } from './MenuButton.svelte'
 export { default as AccountLabel } from './AccountLabel.svelte'
+export { default as TogglableAmountLabel } from './TogglableAmountLabel.svelte'
```

Comment:
Should we create a `labels` directory since there's now more than one + more on the way?

Reply:
Done